src/message/messageActionSheet.js

```diff
@@ -16,18 +20,60 @@ const copyToClipboard = async ({ message, auth }) => {
   Clipboard.setString(rawMessage);
 };
 
-export const actionSheetButtons = [
-  'Narrow to conversation',
-  'Reply',
-  'Copy to clipboard',
-  'Cancel'
+const unMuteTopicHelper = ({ auth, message }) => {
```

Comment:
Definitely not liking 'helper' in names of functions and anything. Usually don't mean anything.